ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -65,6 +65,15 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-assisted
+- as: e2e-metal-assisted-single-node-periodic
```

Comment:
This looks like it's testing BiP specifically, maybe change the name to reflect that?
Or do we not care about non-bip SNO in assisted?

Reply:
assisted-single-node sounds like clear one

Reply:
in this way it's aligned with other jobs